MIP113/MIP113.md

```diff
@@ -238,7 +238,7 @@ This section contains the processes for proposing and settling Atlas Interpretat
 
 #### 2.2.1
 
-Any Maker Governance participant can request an Atlas Interpretation from the Governance Facilitators by submiting a post on the Maker Forum. ​​​​​​​In order for the Governance Facilitators to be bound to comply with the interpretation request, it must be endorsed or supported by at least one AVC Member or Aligned Delegate. In all cases the Facilitator can voluntarily proceed with the interpretation.​​​​​​​
+Any Maker Governance participant can request an Atlas Interpretation from the Governance Facilitators by submiting a post on the Maker Forum. In order for the Governance Facilitators to be bound to comply with the interpretation request, it must be endorsed or supported by at least one AVC Member or Aligned Delegate. In all cases the Facilitator can voluntarily proceed with the interpretation.​​​​​​​
```

Comment:
"submiting" should be corrected to "submitting."

Reply:
Should we remove the mention of "AVC Member"?